### ISSUE TITLE
Considering showing `brand` not `operator` for `amenity=atm` labels

### ISSUE THREAD

**Issue by matkoniecz** [gravitystorm/openstreetmap-carto#4575]
ATM may be often branded with some clear brand like Euronet while being operated by either some franchise, subsidiary, third-party or God know what.

As result mapping `brand` is easy as they are almost always clearly branded while mapping `operator` requires complex research.

### Expected behavior

Map style encourages high quality mapping and good mapping gets typically results better than bad mapping.

So I expect that `brand` would be shown while `operator` not

### Actual behavior

`operator` tag is rendered, `brand` is not

### Links and screenshots illustrating the problem

https://www.openstreetmap.org/node/4736558836
http://overpass-turbo.eu/?q=LyoKVGhpcyBoYcSGYmVlbiBnxI1lcmF0ZWQgYnkgdGhlIG92xJJwxIlzLXR1cmJvIHdpemFyZC7EgsSdxJ9yaWdpbmFsIHNlxLBjaMSsxIk6CsOiwoDCnGFtxI1pdHk9xJRtIGFuxJdixJPFlj0qxZTFlsSfcMSSxJRvciHFmyDEuiBLxJNrw4PCs3fFiMKdCiovCltvdXQ6anNvbl1bdGnFjMW3xbkyNV07Ci8vIGZldMWCxZRyxYAgxYjCnMWqYcWsxa7FsMSbxKvEv8WBxYPEugp7e8SQb2NvZGVBxpNhOsaYxpp3fX0tPi7Gn3LFgsasxYDGicaLxI_ElMSdciDGk3N1bHRzCigKICDGvnF1xJLEmsSjcnTGjcWjOsaVxYnFi8WNxY_FkXTFk8WVxZfFmWTFpseixZ_FoXTFo8WlKsWwx4wgbsapZVsix5xuxY55Ij3HtcegIsW_IsWYx6LHvse0b8WgxJPHqnIiIX4iLirHvijEsMWAxrfFgMa5aMa7YSnGiceNd2F5x7THtse4x7rHvG3Ig8iAx6TIpsiFx6nFo8iKyIzIjsiQyJJhyJTGoMiYyJrHr8aTbMSUacW9yKDFjMe3xY_Io8WSyKbIgcWWyKnIhsWiyInIi8iNyI9dyJHGrcizyJbItcaJyLbGvnDEt27HlseEx4bHiArGhMSYxql5xok-xonJnnNrZcS9cXQ7&c=BR1fnPYO7N
![screen02](https://user-images.githubusercontent.com/899988/173762794-0d431fa5-ccc6-4ddc-bb61-212d94521e30.png)

### Extra

Triggered by https://github.com/streetcomplete/StreetComplete/issues/3782

**Comment by imagico** [comment]
I have doubts this is a good idea:

* `operator` is more than twice as frequently tagged as `brand` on `amenity=atm`.
* `brand` in general and in contrast to `operator` is a rather ill defined concept.  A brand is essentially a string (or graphical symbol) that a certain real world feature is presented with to its user that is not an individual name but a class identifier and that is not generically developed and freely used by society as a whole but made up and/or proprietarized for the purpose of marketing.  That means the idea that any single feature has exactly one brand is often not the case, in particular not for ATMs where the brands advertised on the machines are often what should more accurately be tagged with `network=*`.  For ATMs operated by banks the most prominently advertised brand and the operator are often identical, for ATMs in large shops, airports etc. or other more generic machines - like:

https://en.wikipedia.org/wiki/File:Geldautomaat.jpg
https://en.wikipedia.org/wiki/File:Worn_ATM.jpg
https://en.wikipedia.org/wiki/File:ATM_750x1300.jpg

an attempt to accurately tag `brand=*` will likely mean a semicolon separated list of credit card brands and networks (which we would have to interpret and which - as said - would more accurately belong into the `network=*` tag).

**Comment by matkoniecz** [comment]
Started https://community.openstreetmap.org/t/what-is-the-proper-way-to-tag-visible-company-markings-on-atm-car-charger-etc/104276

**Comment by matkoniecz** [comment]
https://en.wikipedia.org/wiki/File:Worn_ATM.jpg and https://en.wikipedia.org/wiki/File:ATM_750x1300.jpg have no visible ATM branding, what is listed there is supported networks not ATM brand

https://en.wikipedia.org/wiki/File:Geldautomaat.jpg photo is cropping away parts where ATM branding material is usually present

**Comment by Dimitar5555** [comment]
> `operator` is more than twice as frequently tagged as `brand` on `amenity=atm`.

The Name Suggestion Index copies all brand related tags to the corresponding operator tags which likely skews the statistics. It does add both `brand` and `operator` though so eventually they should be almost equal.

For reference: https://github.com/osmlab/name-suggestion-index/blob/main/data/brands/amenity/atm.json#L9-L19

For reference, as of 2025-01-04 there are 158 810 `amenity=atm+operator` and 93 726 `amenity=atm+brand`.

**Comment by imagico** [comment]
> The Name Suggestion Index copies all brand related tags to the corresponding operator tags which likely skews the statistics. It does add both brand and operator though so eventually they should be almost equal.

That would speak against us interpreting either tag.  If mechanical edits have turned several originally semantically distinct tags into a single gray SEO goo then we should ignore all of them.

**Comment by Dimitar5555** [comment]
If Carto rendered `brand` instead of `operator` NSI would likely stop adding `operator` to `amenity=atm`.

**Comment by matkoniecz** [comment]
> brand in general and in contrast to operator is a rather ill defined concept

I would dispute this. As practice in OSM shows it is quite well defined once

And for ATM people keep tagging brand as operator to get it displayed

**Comment by imagico** [comment]
> > brand in general and in contrast to operator is a rather ill defined concept
> 
> I would dispute this. As practice in OSM shows it is quite well defined once

I think i explained and illustrated quite clearly how that is not the case for ATMs.  

> And for ATM people keep tagging brand as operator to get it displayed.

That possibility is one of the reasons why i indicated it might be prudent to stop rendering labels on ATMs altogether.  If a tag that we render as a label turns into a generic label tag (containing whatever the mapper subjectively wants to see labeled rather than a semantically well defined abstract concept like the `operator`) then we have no basis rendering that as a label.

**Comment by dch0ph** [comment]
> That possibility is one of the reasons why i indicated it might be prudent to stop rendering labels on ATMs altogether.

Agree that `brand` is a mushy concept for ATMs. At least from a UK perspective, the brand/operator/network is not an interesting feature to render (it would more useful to indicate whether a fee was charged). It may be more relevant in other countries where the operating bank does make a difference (presumably the original motivation for showing `operator`)? 

`brand` is useful in other contexts, e.g. to indicate the "symbol group" of convenience stores for [QA tools](https://osm.mathmos.net/chains/). But this doesn't mean we should _render_ `brand`.

**Comment by Dimitar5555** [comment]
> * `brand` in general and in contrast to `operator` is a rather ill defined concept.  A brand is essentially a string (or graphical symbol) that a certain real world feature is presented with to its user that is not an individual name but a class identifier and that is not generically developed and freely used by society as a whole but made up and/or proprietarized for the purpose of marketing.

And an `operator` is a string that usually contains the `brand` and some legal form (LLC, PLC ...). So what exactly is the difference, apart from a few extra letters? 

>  That means the idea that any single feature has exactly one brand is often not the case, in particular not for ATMs where the brands advertised on the machines are often what should more accurately be tagged with network=*.
> [en.wikipedia.org/wiki/File:Geldautomaat.jpg](https://en.wikipedia.org/wiki/File:Geldautomaat.jpg)
[en.wikipedia.org/wiki/File:Worn_ATM.jpg](https://en.wikipedia.org/wiki/File:Worn_ATM.jpg)
[en.wikipedia.org/wiki/File:ATM_750x1300.jpg](https://en.wikipedia.org/wiki/File:ATM_750x1300.jpg)

> an attempt to accurately tag brand=* will likely mean a semicolon separated list of credit card brands and networks (which we would have to interpret and which - as said - would more accurately belong into the network=* tag).

Can we not confuse accepted cards and `brand`? Those are two distinct things that can be tagged separately if someone so desires.

> semantically well defined abstract concept like the operator)

George owns a grocery store. He decided to place an ATM in his shop, so he called "Bank Y". "Bank Y" agreed and placed an ATM in his shop. The ATM is branded as "Bank Y" and it is refilled and maintained by "Y Collection Services". You never heard of "Y Collection Services"! Even the ATM has a sticker with text "In case of card withholding or other problems call Bank Y: [phone number]". You found out of the existence of "Y Collection Services" by asking the person who was doing some repair on the ATM the other day.

So, what would you put in `operator` if you were George? And if you were Steve, who randomly noticed that ATM the other day? Would you even bother to look for such information or you will just use the string from the big sticker that says "Bank Y" and put it in  the `brand` tag?

**Comment by dch0ph** [comment]
At the risk of going off topic, there is a similar issue around `amenity=fuel` in the context of petrol/gas stations. These have a "location name" (e.g. "York Road service station") which generally goes into the address rather than the amenity POI. They are typically part of a chain, e.g. "Penny Petroleum", which often goes into `operator` (although arguably it is also a brand). While the fuel sold has a brand e.g. "Esso". It is usually this brand name that goes into `name` because we render `name` for `amenity=fuel`.

To return to the point of ATMs, we have tried to be more specific about what is being rendered in choosing `operator` rather than a generic `name`. But this creates a can of worms with operator vs. brand vs. network. This muddle is arguably hidden for `amenity=fuel` by rendering the generic (if imprecise) `name`. (In my own style, I effectively `COALESCE(name, brand)` for `amenity=fuel`, but this is a hacky / controversial solution.)

TL;DR I think `brand` is a more clearly defined / mappable concept than `operator`, but it is probably a mistake to render either in the context of ATMs.

**Comment by matkoniecz** [comment]
> I think i explained and illustrated quite clearly how that is not the case for ATMs.

If it referring to

> A brand is essentially a string (or graphical symbol) that a certain real world feature is presented with to its user that is not an individual name but a class identifier

then it more or less describes brand but does not present any blockers (`brand` tag to be exact records brand name, which is part of brand mark) 

> and that is not generically developed and freely used by society as a whole but made up and/or proprietarized for the purpose of marketing. 

Also true, though generally irrelevant in this specific case 

> That means the idea that any single feature has exactly one brand is often not the case, 

The same applies to names, operators of given thing and many other attributes

More widely it applies also to surface, shop types and nearly any other attribute.

> in particular not for ATMs where the brands advertised on the machines are often what should more accurately be tagged with network=*. 

Maybe it depends on area? In Poland you have very prominent brand markers and tiny labels, if any, with network info

**Comment by imagico** [comment]
@matkoniecz, @Dimitar5555 - whataboutisms will not help us here.

I had commented on the suitability of `brand=*` as de facto used on ATMs in OSM for our rendering decisions.  If you have evidence from the data that my impression on this is wrong please present that.  If you think my reasoning is flawed then indicate how so.

And yes, the semiotics of brands are an interesting and complex topic, especially on a global, cross cultural level.  I tried to hint at some relatively simple aspects of this in my analysis.

Given the disaster that dumb labeling of `name=*` tags in maps (not only, but also ours here) has contributed to, that we see continuing to unfold these days, and the demonstrated failure of the first attempt at interpreting `brand=*` in this style (#4629) i find the nonchalance with which people suggest starting to undifferentiatedly label that tag in yet another context a bit disturbing.  

Labeling of any mapper supplied tags "as is" in the map is a highly delicate proposition, especially given our international, cross cultural scope.  Don't treat this so lightly.

**Comment by matkoniecz** [comment]
> whataboutisms will not help us here.

whataboutisms is about raising an unrelated issue. In this case your claim was that `brand` is unusable as for some objects multiple brands may be applicable. While such situation is not actually so common for brands, and even if happening it is definitely hardly unique.

**Comment by imagico** [comment]
If you reduce my comments to "there might be features with more than one brand" you are missing the point.

**Comment by matkoniecz** [comment]
then I have no idea what is the point in "That means the idea that any single feature has exactly one brand is often not the case" part if that is not about "there might be features with more than one brand" (or zero of them)

**Comment by imagico** [comment]
The point is that the delineation of discrete brands from the visual design of an ATM or their well-defined representation in the form of a unicode string are not clear in the first place.  Hence the question if 'abcde' is a brand of a certain ATM (even if just one of potentially several) is generally an undecidable question by means of local observation on the ground.  The way some people fond of the `brand=*` tag try to work around this issue is by trying to enforce certain cultural conventions (regarding how to delineate brands and regarding how to represent them in a unicode string in a well defined way) through vehicles like NSI.  But it should be clear that this brings a Wikipedia-like concept of reality into OSM (with all its nasty side effects) and clashes with the traditional value of local verifiability.  And there is no consensus among mappers if this is a desirable approach.

A few more examples of ATMs where the brand would be ill-defined:

https://commons.wikimedia.org/wiki/File:BUGA_2011_193_(5639142750).jpg
https://commons.wikimedia.org/wiki/File:Oktoberfest_2022,_Geldautomat,_1.jpeg
https://commons.wikimedia.org/wiki/File:Wuppertal_D%C3%B6ppersberg_2018_065.jpg

**Comment by matkoniecz** [comment]
> The point is that the delineation of discrete brands from the visual design of an ATM or their well-defined representation in the form of a unicode string are not clear in the first place.

in my experience this is not a substantial problem

at most some people claim that it is idea vs Idea vs IDEA.

**Comment by dch0ph** [comment]
> A few more examples of ATMs where the brand would be ill-defined:

To me, these are examples where no brand is shown and a mapper would leave `brand` (and `operator`) empty. This doesn't make the _concept_ of brand ill-defined. But I suspect it's unusual for an ATM simply to be labelled "ATM". Most banks / operators installing an ATM would surely want to show their branding (this is kind of the definition/function of a brand).

As stated above, I'd be happy to drop labelling of ATMs, as it doesn't seem to serve a useful purpose for the map reader, and isn't really helping the mapper. Showing `<brand> (<operator>)` e.g. "Brand X (Operator Y)", "Brand X", "(Operator Y)", would at least give clearer mapper feedback, but would be further adding to visual clutter IMHO.


> > The point is that the delineation of discrete brands from the visual design of an ATM or their well-defined representation in the form of a unicode string are not clear in the first place.
> 
> in my experience this is not a substantial problem
> 
> at most some people claim that it is idea vs Idea vs IDEA.

I don't see that this problem is particular to `brand` (it applies equally to `name` and `operator`). `brand:wikidata` does work well for normalisation and QA tools, but do we really want/need this for individual ATMs?

**Comment by daganzdaanda** [comment]
It would not bother me if ATMs were not labelled in OSM Carto. They are small features, and the map is cluttered enough in many areas where ATMs are mapped. If there was a zoom level 20 or more, then extra info (fee? brand? network?) might be interesting.

And btw, showing a few examples of ATMs that don't have a brand does not invalidate the tagging of brands for other ATMs. Example image search:
https://duckduckgo.com/?q=geldautomat+sparkasse&iax=images&ia=images
https://duckduckgo.com/?q=geldautomat+VR+Bank&iax=images&ia=images
https://duckduckgo.com/?q=atm+barclays&iar=images&iax=images&ia=images
The "Sparkasse" and "VR Bank" examples would be where "brand" makes a lot of sense. There are a zillion operators for the "Sparkasse" and "VR Bank" brand in Germany. I don't care what local operator is running this, I'm just looking for a red logo or a blue-orange logo to make sure I can get cash without a fee... (But still, we don't have to render the brand IMO)

**Comment by imagico** [comment]
> I'm just looking for a red logo or a blue-orange logo to make sure I can get cash without a fee...

That is indeed the main practical point of displaying something other than the generic ATM symbol (apart from maybe `cash_in=*`).

But tagging wise that information will derive from `operator` at least as frequently as from `brand`.  After all it is ultimately the operator who decides on the fees.

And also this is a good example why flatly labeling a mapper supplied brand string is likely not a good idea.  Because for example it could incentivize mappers tagging [Cash Group](https://en.wikipedia.org/wiki/Cash_Group)/[CashPool](https://en.wikipedia.org/wiki/CashPool) as brand - because that is the practically relevant brand for the ATM user documenting the fees they need to expect.  And they'd not even be verifiably wrong in that since those are indeed brands practically advertised on ATMs.

So far i see quite substantial support for the idea of not labeling ATMs with either `operator` or `brand` any more - which is something i would support.

**Comment by aceman444** [comment]
The amenity=atm wiki only mentions to use 'operator' for the bank operating the atm. There is no mention of 'brand'.
So suggesting to put bank name into 'brand' because it is slapped on big letters on the atm seems weird.
The 'brand' tag is used in different ways, e.g. at car dealers is shows the brands of cars sold, not the brand of the car dealer.
In the past 'brand' on atms could also mean the producer of the atm (the brand of the atm), not the bank operating it (which went into 'operator').
Before this is cleared up, it may really be better to not render anything.

**Comment by matkoniecz** [comment]
>  So suggesting to put bank name into 'brand' because it is slapped on big letters on the atm seems weird.

why? "slapped on big letters on the atm" is a standard case of branding

**Comment by mnalis** [comment]
> why? "slapped on big letters on the atm" is a standard case of branding

That part sounds logical to me, however it is more worrying to me  that:

- https://wiki.openstreetmap.org/wiki/Tag:amenity=atm still does **not** mention `brand=*` as tagging to be used with `amenity=atm` at all (much less in what capacity), even it is tagged at almost half of all ATMs in OSM
- Carto just ignoring that the ATM wiki from the very start (or at last for some last 18-ish years) defined `operator=*` as the _"Name of bank or organisation operating the cash machine."_, and that many people actually tagged according to such documented specs (with `operator` being used on 73%+ of ATMs).

I mean: fine, I was about to make that automated edit for all Croatian ATMs and banks anyway; but it would be nice to at least have ***some*** idea **how** it should be tagged. Over here, we mostly only care about a bank in majority of the cases (at least for 95% of the ATMs who belong to big banks). 

Should bank name then go in `operator` (as wiki currently claims), in `brand` (as it seems Carto prefers), in `network`, or in `name`? Or all (or most?) of the above?

**Comment by Dimitar5555** [comment]
> [wiki.openstreetmap.org/wiki/Tag:amenity=atm](https://wiki.openstreetmap.org/wiki/Tag:amenity=atm) still does **not** mention `brand=*` as tagging to be used with `amenity=atm` at all (much less in what capacity), even it is tagged at almost half of all ATMs in OSM

It is mentioned as of now.

**Comment by dch0ph** [comment]
> Before this is cleared up, it may really be better to not render anything.

This is exactly what PR #5055 did. This is now visible in the recent deployment of version 6.0.0. 

> * Carto just ignoring that the ATM wiki from the very start

Carto generally cheerfully ignores the wiki and endeavours to render tags as mappers are using them.

**Comment by mnalis** [comment]
> It is mentioned as of now.

So, if I understand correctly:

- for major banks, there is consensus that one should set both `operator=*` and `brand=*` (and their `*:wikidata` counterparts) to the name of that Bank (and usually **not** set `network` and `name` / `short_name`)?

- for small independents ATM operator and/or smallish banks, they have their `operator` set to themselves; but if they are part of a network (which they often are - and which significantly influences charges depending on whose card you're using), their `brand` might equal either their `operator` (e.g. ["AuroDomus"](https://nsi.guide/index.html?t=*&k=amenity&v=atm&tt=domus&cc=hr#aurodomus-4d8c03) over here) or their `network` (e.g. ["EuroNet"](https://nsi.guide/index.html?t=*&k=amenity&v=atm&tt=euronet#euronet-d5d850)), depending on what is being prominently displayed in each case?

- there are edge cases when major bank too might in some cases decide to more prominently display global brand then their own bank; e.g. [here](https://www.openstreetmap.org/note/4895849) it would then be `network=Euronet` + `operator=Raiffeisenbank` + brand=`Euronet` ? (there are cases where bank name is even more hidden, but I don't have pics nearby)

- regardless of the above, Carto won't be rendering any name on any ATM, regardless of any tags on it?

Am I getting it (mostly) right? 

If so (or not), should there be some [NSI](https://nsi.guide) revamping too?

(Or is all of this just Carto-specific opinions not matched by the consensus of the rest of OSM world?)

**Comment by aceman444** [comment]
> > [wiki.openstreetmap.org/wiki/Tag:amenity=atm](https://wiki.openstreetmap.org/wiki/Tag:amenity=atm) still does **not** mention `brand=*` as tagging to be used with `amenity=atm` at all (much less in what capacity), even it is tagged at almost half of all ATMs in OSM
> 
> It is mentioned as of now.

I am not sure just putting random changes into the wiki is OK, but if the wiki is about documenting what people do put into the tag, then it would be OK. It is just unclear whether they put it because they want or just because NSI or iD forces them.

**Comment by aceman444** [comment]
> > It is mentioned as of now.
> 
> So, if I understand correctly:
> 
>     * for major banks, there is consensus that one should set both `operator=*` and `brand=*` (and their `*:wikidata` counterparts) to the name of that Bank (and usually **not** set `network` and `name` / `short_name`)?

It does not seem there is any consensus. The wiki now after the change just describes what people do, not what would be good. There was no voting nor we see no links to discussions.

**Comment by aceman444** [comment]
As for rendering the Bank of the atm, it is very important in our country (as some comments indicated it does not matter). But I understand the issue at hand, it is unclear which tag to render, whether brand or operator.

**Comment by dch0ph** [comment]
> * regardless of the above, Carto won't be rendering any name on any ATM, regardless of any tags on it?

That's correct. The overall view was that there wasn't a clear consensus about showing `operator` vs. `brand` and it would be better to show neither.

> (Or is all of this just Carto-specific opinions not matched by the consensus of the rest of OSM world?)

The design philosophy is that Carto should not have opinions and should not be influencing how mappers tag things. Rendering `operator` was seen to be skewing tagging and hence it was better not to render any name-type tags. 

Discussion of how things should be tagged / NSI is really off-topic for this issue tracker.
